### PR TITLE
[zh]sync en/docs/memory-default-namespace.md

### PR DESCRIPTION
**Pull request by youhonglian**
…ault-namespace.md

### CODE REVIEW

**Comment by youhonglian** [comment]
source file：https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/policy/limit-range.md

File: content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 1]
```diff
@@ -247,6 +247,19 @@ resources:
...
+{{< note >}}
+A `LimitRange` does **not** check the consistency of the default values it applies. This means that a default value for the _limit_ that is set by `LimitRange` may be less than the _request_ value specified for the container in the spec that a client submits to the API server. If that happens, the final Pod will not be scheduleable.
+See [Constraints on resource limits and requests](/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests) for more details.
+{{< /note >}}
+-->
+{{< note >}}
+`LimitRange` **不会** 检查它应用的默认值的一致性。
+这意味着由 `LimitRange` 设置的限制默认值可能小于客户端提交给 API 服务器的请求中为容器指定的限制值。
+如果发生这种情况，会导致 Pod 无法被调度。
+详见 [容器的资源限制和请求](/zh-cn/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests)
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:260]:
```suggestion
详见[资源限制和请求的约束](/zh-cn/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests)
```

File: content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 2]
```diff
@@ -247,6 +247,19 @@ resources:
...
+<!--
+{{< note >}}
+A `LimitRange` does **not** check the consistency of the default values it applies. This means that a default value for the _limit_ that is set by `LimitRange` may be less than the _request_ value specified for the container in the spec that a client submits to the API server. If that happens, the final Pod will not be scheduleable.
+See [Constraints on resource limits and requests](/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests) for more details.
+{{< /note >}}
+-->
+{{< note >}}
+`LimitRange` **不会** 检查它应用的默认值的一致性。
+这意味着由 `LimitRange` 设置的限制默认值可能小于客户端提交给 API 服务器的请求中为容器指定的限制值。
+如果发生这种情况，会导致 Pod 无法被调度。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:259]:
```suggestion
如果发生这种情况，最终会导致 Pod 无法被调度。
```

File: content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 3]
```diff
@@ -247,6 +247,19 @@ resources:
...
 
+<!--
+{{< note >}}
+A `LimitRange` does **not** check the consistency of the default values it applies. This means that a default value for the _limit_ that is set by `LimitRange` may be less than the _request_ value specified for the container in the spec that a client submits to the API server. If that happens, the final Pod will not be scheduleable.
+See [Constraints on resource limits and requests](/docs/concepts/policy/limit-range/#constraints-on-resource-limits-and-requests) for more details.
+{{< /note >}}
+-->
+{{< note >}}
+`LimitRange` **不会** 检查它应用的默认值的一致性。
+这意味着由 `LimitRange` 设置的限制默认值可能小于客户端提交给 API 服务器的请求中为容器指定的限制值。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:258]:
```suggestion
这意味着由 `LimitRange` 设置的 **limit** 默认值可能小于客户端提交给 API 服务器的规约中为容器指定的 **request** 值。
```

**Comment by youhonglian** [comment]
@windsonsea Thank you for the modification according to your suggestion

**Pull request closed without merging**